extensions/notion/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "notion",
+  "title": "Notion",
```

Comment:
```suggestion
  "title": "Notion Database",
```

Reply:
As the Extension is meant to do much more thing than just create database entry – shouldn't we just leave Notion?
Or can we change the name later on?

Reply:
You can change the name later on. We want to avoid situations where users have high expectations seeing an extension called "Notion" and then realise that it only adds items to the database.

Reply:
Make sens 👍